packages/config/src/arbitrum-goerli.json

```diff
@@ -77,6 +77,8 @@
   "smoloween_start_block": 735916,
   "smolidays_address": "0xB679eAf002F3F3F1795EA745ab836cf84EB5c85f",
   "smolidays_start_block": 2765235,
+  "smol_chop_shop_address": "0xfb42F170A45D4FE998E62D93Bd3203357daF536B",
+  "smol_chop_shop_start_block": 4014439,
```

Comment:
Please keep the line space separations, they're generally separated by project (i.e. all Smolverse stuff in one block)